src/parser/formatting-evaluator-module.ts

```diff
@@ -91,30 +96,43 @@ export class FormattingEvaluatorModule implements Module {
     const html = this._md.render(comment.content);
     const temp = new JSDOM(html);
     if (temp.window.document.body) {
-      const res = this.classifyTagsWithWordCount(temp.window.document.body);
+      const res = this.classifyTagsWithWordCount(temp.window.document.body, comment.type);
       return { formatting: res };
     } else {
       throw new Error(`Could not create DOM for comment [${comment}]`);
     }
   }
 
-  _countWords(text: string): number {
-    return text.trim().split(/\s+/).length;
+  _countWords(symbols: FormattingEvaluatorConfiguration["multipliers"][0]["symbols"], text: string) {
+    const counts: { [p: string]: { count: number; multiplier: number } } = {};
+    for (const [regex, multiplier] of Object.entries(symbols)) {
+      const match = text.trim().match(new RegExp(regex, "g"));
+      counts[regex] = {
+        count: match?.length || 1,
+        multiplier,
+      };
+    }
+    return counts;
   }
 
-  classifyTagsWithWordCount(htmlElement: HTMLElement) {
-    const tagWordCount: Record<string, { count: number; score: number }> = {};
+  classifyTagsWithWordCount(htmlElement: HTMLElement, commentType: GithubCommentScore["type"]) {
+    const tagWordCount: Record<
+      string,
+      { symbols: { [p: string]: { count: number; multiplier: number } }; score: number }
+    > = {};
     const elements = htmlElement.getElementsByTagName("*");
 
     for (const element of elements) {
       const tagName = element.tagName.toLowerCase();
-      const wordCount = this._countWords(element.textContent || "");
+      const wordCount = this._countWords(this._multipliers[commentType].symbols, element.textContent || "");
```

Comment:
won't this count twice?
for example:
```
<p>this a <span>test</span> string</p>
```
for `p` textContent will be `this a test string` and for `span` it will be `test`

Reply:
Honestly not sure about that one, that is possible. I can make a task to check about it.

Reply:
I've never seen people writing html comments like this. I don't anticipate this scenario to happen normally.

Reply:
> I've never seen people writing html comments like this. I don't anticipate this scenario to happen normally.

Of course people don't write comments in HTML but when markdown gets converted to HTML there might be a scenario where that happens

Reply:
I didn't see this when I authored the first version

Reply:
@0x4007 Actually it happens more than you think I believe, for example:
- some comment with a `tag`

Would contain a `<code/>` within a `<ul/>` and might be counted twice, which is not uncommon to write.

Reply:
I don't understand your example but if there is a code block inside of an unordered list, then there should be credit for both if the config enabled that.

Reply:
Yes but content should not be counted twice, as @whilefoo stated. I don't think it happens but I am not sure either so I should check it.

Reply:
So the strategy is to take the parent or the child element? I suppose the child would be more precise.

Reply:
@0x4007 I guess this will be covered by https://github.com/ubiquibot/conversation-rewards/issues/82